docs/hyperopt.md

```diff
@@ -44,8 +44,9 @@ usage: freqtrade hyperopt [-h] [-v] [--logfile FILE] [-V] [-c PATH] [-d PATH]
                           [--data-format-ohlcv {json,jsongz,hdf5}]
                           [--max-open-trades INT]
                           [--stake-amount STAKE_AMOUNT] [--fee FLOAT]
-                          [-p PAIRS [PAIRS ...]] [--hyperopt-path PATH]
-                          [--eps] [--dmmp] [--enable-protections]
+                          [-p PAIRS [PAIRS ...]] [--hyperopt NAME]
+                          [--hyperopt-path PATH] [--eps] [--dmmp]
+                          [--enable-protections]
```

Comment:
these changes tell me that you're not on the latest develop (which has explicit `--hyperopt` files disabled/hidden).

Best do a merge from upstream and run this command again ...

Reply:
Thank you for letting me know. I am a junior developer trying to figure this stuff all out. I believe I made the merge correctly. Hopefully that takes care of the issue.

Reply:
i guess it will - if you run `freqtrade hyperopt --help` again and copy/paste the content in here ... :)